src/main.js

```diff
@@ -85,7 +85,7 @@ const createWindow = () => {
   });
 
   // Uncomment this to open the Chrome DevTools
-  // mainWindow.webContents.openDevTools();
+  mainWindow.webContents.openDevTools();
```

Comment:
comment and re-commit, please

Reply:
weird, I thought I push it to another remote so that it would not mess up this branch. But I'm still in the progress of debugging a cache issue so this PR should be closed.